alien4cloud/alien4cloud_structs.go

```diff
@@ -329,6 +329,12 @@ type DeploymentArtifact struct {
 	Description          string                 `json:"description,omitempty"`
 }
 
+type ListTopologiesCreateRequest struct {
```

Comment:
This new type may not be needed as the type searchRequest described above is identical

Reply:
Yes, I agree. Done.